packages/frontend/@n8n/design-system/src/css/_tokens.scss

```diff
@@ -662,6 +662,7 @@
 	--font-line-height-regular: 1.3;
 	--font-line-height-loose: 1.35;
 	--font-line-height-xloose: 1.5;
+	--font-line-height-xxloose: 1.6;
```

Comment:
Let's not add new css tokens without discussing it with broader audience :). It's fine to use 1.6 as an exception in TextMessage.vue